client/src/ui/PageFooter.tsx

```diff
@@ -90,77 +107,35 @@ export const PageFooter = () => {
       <div
         style={{
           display: "flex",
-          alignItems: "center",
+          justifyContent: "flex-end",
+          gap: "100px", // Space between icons
         }}
       >
-        <Link href="#">
-          <Image
-            src="/x-logo/X-logo-black.png"
-            alt="X"
-            width={24}
-            height={24}
-            style={{
-              margin: "0 40px",
-              opacity: 0.7,
-              transition: "opacity 0.3s ease",
-            }}
-            onMouseEnter={(e) => (e.currentTarget.style.opacity = "1")}
-            onMouseLeave={(e) => (e.currentTarget.style.opacity = "0.7")}
-          />
-        </Link>
-        <Link href="#">
-          <Image
-            src="/Instagram-logo/Instagram_Glyph_Black.svg"
-            alt="Instagram"
-            width={24}
-            height={24}
-            style={{
-              margin: "0 40px",
-              opacity: 0.7,
-              transition: "opacity 0.3s ease",
-            }}
-            onMouseEnter={(e) => (e.currentTarget.style.opacity = "1")}
-            onMouseLeave={(e) => (e.currentTarget.style.opacity = "0.7")}
-          />
-        </Link>
-        <Link href="#">
-          <Image
-            src="/LinkedIn-logo/In-Black.png"
-            alt="LinkedIn"
-            width={24}
-            height={24}
-            style={{
-              margin: "0 40px",
-              opacity: 0.7,
-              transition: "opacity 0.3s ease",
-            }}
-            onMouseEnter={(e) => (e.currentTarget.style.opacity = "1")}
-            onMouseLeave={(e) => (e.currentTarget.style.opacity = "0.7")}
-          />
-        </Link>
-        <Link href="#">
-          <Image
-            src="/Meta-logo/Meta-logo.png"
-            alt="Meta"
-            width={24}
-            height={24}
-            style={{
-              margin: "0 40px",
-              opacity: 0.7,
-              transition: "opacity 0.3s ease",
-            }}
-            onMouseEnter={(e) => (e.currentTarget.style.opacity = "1")}
-            onMouseLeave={(e) => (e.currentTarget.style.opacity = "0.7")}
-          />
-        </Link>
+        {images.map((image, index) => (
+          <Link key={index} href={image.route}>
+            <Button
+              sx={{
+                minWidth: "auto", // Remove the default button minWidth
+                padding: 0, // Remove padding
+                opacity: 0.7,
+                transition: "opacity 0.3s ease",
```

Comment:
I would remove this transition since it looks kind of janky for some of the icons
<img width="96" alt="image" src="https://github.com/user-attachments/assets/af1f3ebd-d12e-4120-8926-8f0936a0f13d">